src/sources/forus-webshop/pug/tpl/directives/mobile-footer.pug

```diff
@@ -0,0 +1,21 @@
+.block.block-sticky-footer.show-sm(ng-class='{ "scrolled" : !visible }')
+    ul.sticky-footer-list
+        li.sticky-footer-item(ui-sref="providers" ui-sref-active="active")
+            include ../../../resources/_webshop-common/assets/img/icon-menu/icon-menu-providers.svg
+            span(translate="topnavbar.buttons.providers")
+
+        li.sticky-footer-item(ui-sref="products" ui-sref-active="active")
+            include ../../../resources/_webshop-common/assets/img/icon-menu/icon-menu-products2.svg
+            span(translate="topnavbar.buttons.products")
+
+        li.sticky-footer-item(ng-if="!$root.auth_user" ng-click="openAuthCodePopup()" ui-sref-active="active")                     
```

Comment:
@GerbenBosschieter this one should work the same as the start button. Now it always opens AuthCodePopup()

Reply:
I will first do all the refactoring and than fix this. Not sure if I will manage, but will request your support if it's needed.